discovery-infra/download_logs.py

```diff
@@ -137,10 +150,11 @@ def download_logs(client: InventoryClient, cluster: dict, dest: str, must_gather
         with suppressAndLog(AssertionError, ConnectionError, requests.exceptions.ConnectionError):
             client.download_metrics(os.path.join(output_folder, "metrics.txt"))
 
-        for cluster_file in ("bootstrap.ign", "master.ign", "worker.ign", "install-config.yaml", "custom_manifests.yaml"):
-            with suppressAndLog(assisted_service_client.rest.ApiException):
-                client.download_and_save_file(cluster['id'], cluster_file,
-                                              os.path.join(output_folder, "cluster_files", cluster_file))
+        download_and_save_files(["bootstrap.ign", "master.ign", "worker.ign", "install-config.yaml"],
+                                client, cluster, output_folder)
+
+        # best effort
+        download_and_save_files(["custom_manifests.yaml"], client, cluster, output_folder, raise_exception=False)
```

Comment:
I think that actually there are more files that aren't necessarily available.. for example if there are no workers there won't be an ignition file.

It comes back to the point where the service should provide a list of all the avilable files to be downloadef
cc @osherdp